docs/blog/2023-06-01-drand-explainer.md

```diff
@@ -0,0 +1,107 @@
+
+Public random numbers are a little bit different. These are numbers we _want_ everybody to see: think lottery numbers, the roll of a dice in a board game, selecting a business to be audited at random or a coin flip for who takes the kick-off in a football match.
+This is exactly the type of randomness that drand provides - you definitely shouldn't generate your Bitcoin private key using drand (or if you do, please tell us so we can liberate all your Bitcoin to help fund the team!).
+
+## Verifiability
+
+In the examples of public randomness we discussed above, humans have come up with a myriad of dances to ensure fairness: lottery numbers are drawn from fancy machines built by companies dedicated to fairness, dice are shared in board games to ensure that a single player isn't using a rigged die, and a trusted third party (the referee) picks the coin and flips it to decide which team takes the kick-off.
+
+None of these solutions are really _verifiable_, but we've got established procedures around them that increase our assurance that they're fair.
+
+For generating random numbers fast and at scale however, it's a lot more difficult to prove fairness:
+- if I trust a third party to generate the random number, how do I know they really chose it randomly?
+- if I trust a third party to run a random number generator I have audited, how do I know they're running the code they say they are?
+- if I run some code to generate a random number, how do I know the code is really statistically random (and bug free!)?
+
+Human intuition can play tricks on us when it comes to randomness. If you were to look at the following binary numbers, which do you think is the most random?
+
+```
+1111111111111111
+0000000000000000
+1001011010001001
+```
+
+At first glance the first two seem far too uniform to be random, but from a random selection of values from `0000000000000000` to `1111111111111111` (inclusive), they are all equally likely to occur!
+
+How then can we verify a randomly selected number was really randomly selected? It seems impossible. However in drand, we exploit some base cryptographic principles to make this possible.  To fully understand it though, we will have to do a little background work.
+
+## Digital signatures
+
+Digital signatures are a lot like human signatures: we take some piece of data (like a contract or a letter) and we append a message to it that uniquely identifies us and binds the signature to the piece of data.
+That said, human and digital signatures differ in a few ways:
+- human signatures map to a single person; digital signatures map to a single private key (and a person could own multiple private keys)
+- human signatures can be copied to another piece of data and still be valid; digital signatures are bound to a single piece of data (as the data is 'included' in the signature in a mathematical sense)
```

Comment:
Technically they are not bound to a single piece, it's just very unlikely to be able to find another piece that collisions with it.

Reply:
updated